crate/utils/src/crypto/certificate/mod.rs

```diff
@@ -1 +1 @@
-pub mod attributes;
+//pub mod attributes;
```

Comment:
Might also want to delete certificate directory at this point ?

Reply:
done in 2b1a3c262d233c02f86172ada0a02f0df3ef1fbf